facets.wdl

```diff
@@ -41,38 +45,48 @@ workflow facets_workflow {
             min_read_counts_tumor=min_read_counts_tumor,
             pseudo_snps=pseudo_snps,
             memoryGB=pileup_memoryGB,
-            diskGB=pileup_diskGB
+            diskGB=pileup_diskGB,
+            preemptible_attempts=preemptible_attempts
     }
 
     call FACETS {
         input:
             pair_name=pair_name,
             pileup=Pileup.pileup,
+            ndepth=min_read_counts_normal,
             cval=cval,
             maxiter=maxiter,
+            seed_initial=seed_initial,
+            seed_iterations=seed_iterations,
             memoryGB=facets_memoryGB,
-            diskGB=facets_diskGB
+            diskGB=facets_diskGB,
+            preemptible_attempts=preemptible_attempts
     }
 
     call InferWGD {
         input:
             cncf=FACETS.cncf,
             memoryGB=inferwgd_memoryGB,
-            diskGB=inferwgd_diskGB
+            diskGB=inferwgd_diskGB,
+            preemptible_attempts=preemptible_attempts
     }
 
     output {
         File facets_pileup = Pileup.pileup
         File facets_genome_segments = FACETS.genome_segments
         File facets_diagnostics_plot = FACETS.diagnostics_plot
```

Comment:
Please just double check this is being produced. For whatever reason, when I ran with Jake's old WDL, these files were spuriously small and could not be rendered.

Reply:
Which files in particular?

Reply:
When the files were just super small, was it for cases that had NA purity calls? If that is the case, then the PDFs may not have been produced and may have just been preallocated as WDL / Cromwell can't handle optional outputs.

Reply:
I'm specifically referring to the diagnostic plots, which were 11B unrenderable files in all cases for me, even when there was a purity/ploidy solution. I think if in your test runs you're getting an actual figure, then the problem doesn't apply anymore.

Reply:
Oh weird. I've only observed that when purity is NA. Let's look into it if it comes up again.